src/springfield/entity.py

```diff
@@ -105,9 +107,11 @@ def jsonify(self):
         Return a dictionary suitable for JSON encoding.
         """
         data = {}
-        for key, val in self.__values__.iteritems():
```

Comment:
Once the object is created i think we don't have a case to reset/populate the default values. I'm in favor of using ```_defaults=True``` in constructor  instead of ```populate_defaults()``` method. For sparse updates the user will create objects without setting the ```_defaults``` parameter.

deviating from the current topic of discussion. Entity/Person doesn't have a way to specify the fields that should not appear in return value of jsonify(). Consider the following example. 
````
class User(Entity):
     username = fields.String()  
     password = fields.String()
``` 
Let's say that the user is resetting the password. He enters the current password and the new password. the api would get the password from the db, validate the current-password given by the user, update the new password. The User() object will have the password field set. If the response is expected to be user.jsonify() it will contain the password field.  Is there a plan to handle such cases either using Personas or in Entity/Fields?